metadata-node-red-app

```diff
@@ -0,0 +1,99 @@
+{
+    "id": "node-red-app",
+    "crn:v1:staging:public:resource-catalog::a/bef71ea83f3d0c18eb701256cf445eb3::template:node-red-app",
+    "url": "https://globalcatalog.test.cloud.ibm.com/api/v1/node-red-app?complete=true",
```

Comment:
this url points to the testing environment because it has test in the url can you fix that, check out https://github.ibm.com/arf/starter-web-basic/ to compare

Reply:
Dilan-- since this is the master branch, this metadata need to point to the production catalog. We could put this in the test/staging environment with that catalog URL, but we'd need a `stage1` branch of the app.

Reply:
Hey @nfstein I checked out the metadata for the web basic ones and they all use the same format for the url, which one were you suggesting I look at?

Reply:
https://github.ibm.com/arf/starter-web-basic/ master branch